### DISCUSSION TITLE
Multiple Datasources

### DISCUSSION THREAD

**Discussion by ITconsulto** [Budibase/budibase#9791] [Help] [answered]
Is it possible to synchronize multiple data sources or, at least, transfer some of the data/tables to another data source?
An example would be trying to copy the table in the Budibase DB data source to Google Sheets.

**Comment by andz-bb** [comment, marked as answer]
hey @ITconsulto, the best way to do this would be to create Automations that can write data to your secondary source. You can set these to trigger on row create/update/delete and then add a step to run the equivalent action on the other table. Using bindings you can assign the data from the trigger to the appropriate fields on the other table. Hope this helps, if you have any questions on this let me know and I can show you in more detail!

**Reply by ITconsulto** [reply]
hey @andz-bb thanks for the info. could you provide me more details please? An step by step example would be great. Maybe we could try to add a row from a table in budibase DB to a Table in Google Sheet. That seems quite complicated. Maybe you can help me out.
